provider/datagen/src/tests/make_testdata.rs

```diff
@@ -46,6 +47,15 @@ fn generate_json_and_verify_postcard() {
         ),
     });
 
+    let stubdata_out = Box::new(BakedStubdataExporter(
+        baked_exporter::BakedExporter::new("tests/data/stub".into(), {
```

Comment:
This is not only useful for build times, but also for debugging baked output. With that in mind, please
* change the path to `tests/data/baked`
* set `pretty`
* move the VSCode instructions to `CONTRIBUTING.md` or somewhere else. They're not relevant for this test
* consider not filtering the `und` payload if size allows it. We're already including singletons anyway.